lightwood/mixers/nn/nn.py

```diff
@@ -27,8 +28,14 @@ def __init__(self):
 
         self.criterion = nn.MSELoss()
         self.epochs = 120000
-        self.optimizer_class = optim.Adadelta
-        self.optimizer_args = {'lr': 0.1}
+
+        if self.dynamic_adamw:
```

Comment:
this will never check True, as the class is initialized and then self.*<-->config attributes are mapped.

Reply:
Ah yes, I know, I left it in there for now so that I can play around with it (and maybe ask other people to enable it for internal testing). I want to read a bit more about how to adapt the learning rate and how to get very-fast convergence using adamw before I expose it to normal users and allow it to be enabled via config or based on the data.